chef/cookbooks/barclamp/libraries/nic.rb

```diff
@@ -751,6 +764,42 @@ def remove_slave(slave)
       ::Kernel.system("ovs-vsctl del-port #{@nic} #{slave}")
       slave.down
     end
+
+    def up
+      slaves.each{ |s|s.up }
```

Comment:
Style/SpaceAroundBlockParameters: Space after closing `|` missing.<br>Style/SpaceBeforeBlockBraces: Space missing to the left of {.<br>Style/SymbolProc: Pass `&:up` as an argument to `each` instead of a block.